packages/reports/addon/components/common-actions/schedule.ts

```diff
@@ -136,7 +136,7 @@ export default class ScheduleActionComponent extends Component<Args> {
    */
   _createNewDeliveryRule() {
     return this.store.createRecord('delivery-rule', {
-      format: { type: this.formats.firstObject },
+      format: this.formats.firstObject,
```

Comment:
should we have a return type for this method?

Reply:
Looks like `DeliveryRule` has some incorrect TS saying this is a string ([WS DeliveryRule Model](https://github.com/yavin-dev/framework/blob/71358ed71cbcf012bb6d97e744abc2076b7b44a6/packages/webservice/models/src/main/kotlin/com/yahoo/navi/ws/models/beans/DeliveryRule.kt#L84) says `{ format: { type: 'csv' }}` is correct.) We should make a `delivery-format` fragment like we do for [scheduling-rules fragment](https://github.com/yavin-dev/framework/blob/master/packages/core/addon/models/fragments/scheduling-rules.ts) and use it like this in the [delivery-rule model](https://github.com/yavin-dev/framework/blob/71358ed71cbcf012bb6d97e744abc2076b7b44a6/packages/core/addon/models/delivery-rule.ts#L47-L48)